### PR TITLE
Db

### PR DESCRIPTION
**Pull request by xanzadu**
Update grid of images with multiple changes:
-Proper spacing added
-Background shadow on image added
-More appealing text
-Text size changes to reflect importance of properties
-Add to cart button on images (currently not functional)
-Changed api call to get cart and farmers for images

### CODE REVIEW

**Comment by ykim9611** [comment]
I was able to see the "+" button on each product and by comparing this branch with the main branch, I was able to confirm the difference made to each item as mentioned in your comments above. Other than the comment on the console.log, all looks good!

File: server/controllers/shopController.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ const pool = require('../../db/index');
...
   const { category } = req.params;
   const query = `
-    SELECT * FROM products WHERE categories LIKE '%${category}%'
+    SELECT products.product_id AS key_product_id, * FROM products LEFT JOIN farmers ON products.farmer_id = farmers.farmer_id LEFT JOIN cart ON products.product_id = cart.product_id WHERE categories LIKE '%${category}%'
   `;
 
   try {
     const result = await queryDB(pool, query);
-
+    console.log('server', result.rows);
```
Comment (ykim9611) [inline server/controllers/shopController.js]: After testing the result with console.log, we should ideally remove it before pushing it up to GitHub.

**Review by ykim9611 — DISMISSED** [review]
The console.log can removal can be done at a later date. Looks good!

File: server/controllers/shopController.js (hunk 1 above)
Comment (xanzadu) [reply server/controllers/shopController.js]: Deleted

**Review by jessicaawhy — DISMISSED** [review]
Verified latest commit removes unnecessary logging, approved.

File: client/components/Shop/Products/ProductGrid.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              Farms
+            </FarmDiv>
+          </div>
+          <ProductDiv>{product.product_name}</ProductDiv>
+          <PriceSpan>
+            <span>{product.quantity}</span>
+            <span>
+              $
+              {product.price}
+              .99
```
Comment (itsnormsyo) [inline client/components/Shop/Products/ProductGrid.jsx:84]: Everything ends with $.99 cents. Most excellent.. Our customers will think they are getting a sweet deal and will be stuck on our page hunting more excellent deals.

**Review by itsnormsyo — APPROVED** [review]
Product Grid Component looks good. Seems straight forward. Most Excellent

**Pull request merged**